File: back/src/main/java/com/bank/account/dto/AccountBankResponseDTO.java
package com.bank.account.dto;

import lombok.*;

import java.time.LocalDateTime;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@Builder
@EqualsAndHashCode
public class AccountBankResponseDTO {
    Long id;
    ClientResponseDTO client;
    private Long solde;
    private Long decouvert;
    LocalDateTime dateCreation;
}

File: front/src/interface/accountBankResponseDTO.ts
import { ClientResponseDTO } from "./clientResponseDTO";

export class  AccountBankResponseDTO {
    public id?: number;
    public solde?: number;
    public decouvert?: number;
    public client?: ClientResponseDTO;
    public dateCreation?: Date;
    constructor (solde: number, decouvert: number){
        this.solde = solde;
        this.decouvert = decouvert;
    }
}